twilight-lavalink/src/model/outgoing.rs

```diff
@@ -0,0 +1,438 @@
+//! Events that clients send to Lavalink.
+use serde::{Deserialize, Serialize};
+use twilight_model::{
+    gateway::payload::incoming::VoiceServerUpdate,
+    id::{marker::GuildMarker, Id},
+};
+
+/// The track on the player. The encoded and identifier are mutually exclusive.
+/// Using only encoded for now. Encoded was chosen since that was previously
+/// used in the v3 implementation. We don't support userData field currently.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct UpdatePlayerTrack {
+    /// The base64 encoded track to play.
+    pub encoded: Option<String>,
+}
```

Comment:
Can you add support for `identified`? I think we can leave `userData` out for now.